runtimes.json

```diff
@@ -1,671 +1,726 @@
 {
-    "aliases": {
-        "latest": "0.3.7-R2-2-1",
-        "openmp": "openmp-beta-v10",
-        "0.3.DL": "0.3.DL-R1",
-        "0.3.7": "0.3.7-R2-2-1",
-        "0.3z": "0.3z-R4",
-        "UGMP": "UG-MP"
+    "aliases":{
```

Comment:
Please don't re-format the entire document when adding a new runtime link.

Reply:
Alright, but there are some trailing spaces where they shouldn't.

Reply:
worth doing a separate PR with a Prettier.js pass on the JSON 👍